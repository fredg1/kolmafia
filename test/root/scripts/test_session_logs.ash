print("2000/01/01, +2");
session_logs("heeheehee", "20000101", 2).dump();

print("");
print("2000/01/01, -2");
session_logs("heeheehee", "20000101", -2).dump();
