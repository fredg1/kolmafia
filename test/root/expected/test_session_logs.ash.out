2000/01/01, +2
aggregate string [3]
0 => 20000101
1 => 20000102
2 => 20000103

2000/01/01, -2
aggregate string [3]
0 => 19991230
1 => 19991231
2 => 20000101
